code/modules/reagents/reagent_containers/glass/jar.dm

```diff
@@ -0,0 +1,129 @@
+/obj/item/weapon/reagent_containers/glass/jar
+	name = "jar"
+	desc = "A large jar."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "jar"
+	amount_per_transfer_from_this = 10
+	possible_transfer_amounts = list(5,10,15,25,30)
+	flags = FPRINT  | OPENCONTAINER
+	volume = 250
+	starting_materials = list(MAT_GLASS = 1000)
+	w_type = RECYK_GLASS
+	w_class = W_CLASS_MEDIUM
+	melt_temperature = MELTPOINT_GLASS
+	origin_tech = Tc_MATERIALS + "=1"
+	var/obj/held_item = null
+
+/obj/item/weapon/reagent_containers/glass/jar/New()
+	..()
+	processing_objects.Add(src)
+
+/obj/item/weapon/reagent_containers/glass/jar/Destroy()
```

Comment:
You need to remove it from processing objects.

Reply:
removed from processing objects